openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -56,19 +49,6 @@ def create(self, subset_name, instance_data, pre_create_data):
 
             self._update_tool_with_data(saver, data=instance_data)
 
-            saver["OutputFormat"] = file_format
-
-            # Check file format settings are available
-            if saver[file_format] is None:
-                raise RuntimeError(
-                    f"File format is not set to {file_format}, this is a bug"
-                )
-
-            # Set file format attributes
-            saver[file_format]["Depth"] = 0  # Auto | float16 | float32
-            # TODO Is this needed?
-            saver[file_format]["SaveAlpha"] = 1
```

Comment:
This PR does mean that we're now losing these defaults? Could you identify what this means? What is the default depth for e.g. EXR format without this? Is it also `0`? 
And how about alpha saving? Does it also default to `1`?

Just want to clarify what this influences outside of the scope of just adding extra extension formats.

You state:
> I've removed some potential bloat saver["OutputFormat"] = file_format. In my testing Fusion is able to infer this information based on the file extension (Fusion17 and 18).

So does this mean that **exr** means: EXR Format 0 (auto) and Save Alpha (1)?

Reply:
- OutputFormat - Fusion is able to infer this based on the provided path. So in the case of EXR it changes to "OpenEXR Files" and TGA - "Targa Files"
- Depth - Fusion Savers have a default of Depth = 0 so that code was already redundant 
- SaveAlpha - This doesn't seem to have a purpose, it doesn't exist on the Saver at any stage by default, unless we're using this elsewhere?

Reply:
> SaveAlpha - This doesn't seem to have a purpose, it doesn't exist on the Saver at any stage by default, unless we're using this elsewhere?

Ah, then maybe this is the case only for PNG.